ha/resource/sspl

```diff
@@ -115,22 +115,50 @@ stateful_check_state() {
     return 7
 }
 
+start_sspl_service() {
+        systemctl start sspl-ll
+        echo "START (master): $?"
+        count=0
+        while true
+        do
+            status=`systemctl is-active sspl-ll`
+            if [[ $status == "active" ]]; then
+                return 0
+            elif [[ $status == "failed" ]]; then
+                return 1
+            elif [[ $status == "unknown" ]]; then
+                ((count++))
+                if [ $count == 5 ]; then
+                    return 1
+                fi
+            fi
+            sleep 1
+        done
+}
+
 stateful_start() {
     stateful_check_state "master"
     if [ $? -eq 0 ]; then
         # CRM Error - Should never happen
         log "START (master)..."
-        systemctl start sspl-ll
-        log "START (master): $?"
-        return $OCF_RUNNING_MASTER
+        start_sspl_service
+        if [ $? == 0 ]; then
+            return $OCF_RUNNING_MASTER
+        elif [ $? == 1 ]; then
+            return $OCF_FAILED_MASTER
+        else
```

Comment:
return code is either 0 or 1, it will never reach here.

Reply:
Yes, Not required elif condition. Updated in the latest patch